set.mm

```diff
@@ -62783,17 +62788,33 @@ ordered pairs (for use in defining operations).  This is a special case
 
   ${
     fvoveq1d.1 $e |- ( ph -> A = B ) $.
-    $( Equality deduction for nested function and operation value.
+    $( Equality deduction for nested function and operation value.  Technical
```

Comment:
It is an accepted idea to introduce an infrastructure theorem for the sole purpose of extracting repeated proof steps.  I did it my self on various occasions (see sylnbi and many many more).  This is covered by file:///home/wolf/metamath/web/conventions.html:

> Usually we will also automatically accept a _new_ theorem that is used to shorten multiple proofs, if the total size of set.mm (including the comment of the new theorem, not including the acknowledgment) decreases as a result.

As a rule of thumb, Norm accepted one, if it was referenced at least seven times.  fvop1e1 is applied 20 times.  To best match the above rule, I would get rid of the second sentence (Technical...).  It wasn't considered necessary in the past, and I am not fond of starting this habit.

Reply:
I agree with @wlammen.

Reply:
I would agree for theorems like ~fvoveq1 (or ~fvoveq2, see above) and ~2fveq3 because they are self-explanatory. But what about 

```
ovanraleqv.1 $e |- ( B = X -> ( ph <-> ps ) ) $.
ovanraleqv $p |- ( B = X -> ( A. x e. V ( ph /\ ( A .x. B ) = C ) <-> A. x e. V ( ps /\ ( A .x. X ) = C ) ) )
```
and 
```
fv0p1e1 $p |- ( N = 0 -> ( F ` ( N + 1 ) ) = ( F ` 1 ) )?
```
Nobody who would stumble over these theorems could understand their meanings. They are really technical only. Things may be different for first-order logic  where a lot of varaitions of the same "idea" are provided as theorems (like ~sylnbi, which I would see in the same group of infrastructure theorems as ~2fveq3). Mathematically, theorems like ~ovanraleqv and ~fv0p1e1 do not make any sense.

As a compromise, we may make such theorems lemmas (~ovanraleqv and ~fv0p1e1 -> ~ovanraleqvlem and ~fv0p1e1lem), but this would still require an explanation in the comment, and it increases the (textual) size of the proofs unnecessarily.

Reply:
I removed "Technical theorem to be used to reduce the size of a significant number of proofs." from the comments except for ~ovanraleqv and ~fv0p1e1.

Reply:
My justification is: the fact that they are technical lemmas is obvious from their statement, and the fact that they are used in many theorems appears on the webpage ("this theorem is referenced by"). So the comment "Technical theorem..." is not needed. It does not hurt either, so no strong opinion.